include/fizzy/fizzy.h

```diff
@@ -213,6 +213,15 @@ uint8_t* fizzy_get_instance_memory_data(FizzyInstance* instance);
 /// @note    Function returns memory size regardless of whether memory is exported or not.
 size_t fizzy_get_instance_memory_size(FizzyInstance* instance);
 
+/// Find exported global by name.
+///
+/// @param  instance        Pointer to instance.
+/// @param  name            The global name. NULL-terminated string. Cannot be NULL.
+/// @param  out_global      Pointer to output struct to store found global. Cannot be NULL.
```

Comment:
Is there any better best practice than prefixing it with `out_`?

Reply:
I'm not aware of any.

Reply:
In most case we can come up with a sentinel values. E.g. here you can have 
```
FizzyExternalGlobal fizzy_find_exported_global(
    FizzyInstance* instance, const char* name);
```

And then "not-found" can be represented by `FizzyExternalGlobal{nullptr, FizzyValueTypeVoid}`.